src/uikit.cs

```diff
@@ -35171,6 +35982,42 @@ interface UISceneWindowingBehaviors {
 		bool Miniaturizable { [Bind ("isMiniaturizable")] get; set; }
 	}
 
+	[iOS (26, 0), TV (26, 0), MacCatalyst (26, 0)]
+	[BaseType (typeof (NSObject))]
+	[DisableDefaultCtor]
+	interface UISceneDestructionCondition : NSCopying {
+
+		[NoiOS, NoTV, NoMacCatalyst]
+		[Static]
+		[Export ("userInitiatedDismissal")]
+		UISceneDestructionCondition UserInitiatedDismissal ();
+
+		[NoiOS, NoTV, NoMacCatalyst]
+		[Static]
+		[Export ("systemDisconnection")]
+		UISceneDestructionCondition SystemDisconnection ();
```

Comment:
This is a property in Swift: https://developer.apple.com/documentation/uikit/uiscene/destructioncondition/systemdisconnection